### PR TITLE
Feature/calc pricing service include diff

### PR DESCRIPTION
**Pull request by mkurek**
* improved pricing service plugin selection; renamed ceilometer plugin to fixed price plugin
* added charging pricing service by difference between real and calculated costs (ex. from fixed prices)

### CODE REVIEW

File: src/ralph_scrooge/models/usage.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from django.db import models as db
 from django.utils.translation import ugettext_lazy as _
 
+
```
Comment (kula1922) [inline src/ralph_scrooge/models/usage.py]: Needless line

File: src/ralph_scrooge/plugins/cost/pricing_service.py [hunk 2]
```diff
@@ -504,6 +514,56 @@ def _get_service_dynamic_extra_cost(
...
+                ps_real_cost = plugin_runner.run(
+                    'scrooge_costs',
+                    'pricing_service_plugin',
+                    **args
+                )
+
+                # calculate difference between real cost and
+                diff = ps_real_cost - ps_cost
+                result[ps.id] = diff, {}
+            except (KeyError, AttributeError):
```
Comment (kula1922) [inline src/ralph_scrooge/plugins/cost/pricing_service.py]: wrap only code where except conditions could be happen

File: src/ralph_scrooge/plugins/cost/pricing_service.py [hunk 3]
```diff
@@ -504,6 +514,56 @@ def _get_service_dynamic_extra_cost(
...
+        result = {}
+        for ps in pricing_service.charged_by_diffs.all():
+            try:
+                args = dict(
+                    type='total_cost',
+                    date=date,
+                    forecast=forecast,
+                    pricing_service=ps,
+                    collapse=True
+                )
```
Comment (kula1922) [inline src/ralph_scrooge/plugins/cost/pricing_service.py]: why not args = {...} ? is there any different?

Comment (mkurek) [inline src/ralph_scrooge/plugins/cost/pricing_service.py]: why not dict(...) if there is no difference? :)

**Pull request merged**